.gitignore

```diff
@@ -127,3 +128,6 @@ dmypy.json
 
 # Pyre type checker
 .pyre/
+
+# VSCode settings
+.vscode
```

Comment:
Final new line?

Reply:
I've definitely gotta add that to my IDE formatter like black I keep missing those! 
My bad!